### ISSUE TITLE
How do you train a lot of ids

### ISSUE THREAD

**Issue by guanyonglai** [whut2962575697/naic_reid#5]
I use your code to train a lot of ID, but convergence is very slow

**Comment by whut2962575697** [comment]
Since we use warmup, the convergence is slow at the beginning, but it can converge in the end. Maybe you can try to increase the training epochs or increase the learning rate
